xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+        </para>
+</warning>
+</sect1>
+
+<sect1 xml:id="sec.openvswitch">
+<title>Install Openvswitch on Control and Compute Nodes</title>
+<para>
+    Install Open vSwitch on your control and compute nodes, version 2.4.1 or greater. The first command installs the necessary packages, and the second command verifies the version number.
+</para>
+<screen><prompt>root # </prompt><command>zypper in openvswitch openvswitch-switch openvswitch-kmp-default</command>
+<prompt>root # </prompt><command>modinfo openvswitch</command>
+</screen>
+
+<para>
+    Create the required OVS (open virtual switch) bridges. Run these commands on the control node:
+</para>
+<screen><prompt>root # </prompt><command>systemctl start openvswitch</command>
+<prompt>root # </prompt><command>systemctl enable openvswitch</command>
+<prompt>root # </prompt><command>ovs-vsctl add-br br-int</command>
+</screen>
+
+<para>
+    On the compute nodes you will also create <literal>vxlan</literal> ports.
+</para>
+
+<screen><prompt>root # </prompt><command>systemctl start openvswitch</command>
+<prompt>root # </prompt><command>systemctl enable openvswitch</command>
+<prompt>root # </prompt><command>ovs-vsctl add-br br-int</command>
+<prompt>root # </prompt><command>ovs-vsctl set bridge br-int protocols=[]</command>
+<prompt>root # </prompt><command>ovs-vsctl add-port br-int br-int_vxlan0 \
+    -- set Interface br-int_vxlan0 type=vxlan options:remote_ip=flow \
+    options:key=flow options:dst_port=8472</command>
+</screen>
+
+<para>
+    Run <command>lsmod</command> to verify that the <literal>vxlan</literal> port was created.
+</para>
+
+<screen><prompt>root # </prompt><command>lsmod | grep openvswitch</command>
+openvswitch   130813  1 vport_vxlan</screen>
+</sect1>
+
+<sect1 xml:id="sec.barclamps">
+<title>Deploy Barclamps</title>
+<para>
+    Deploy all pertinent OpenStack barclamps using your Crowbar Web interface. This depends on your own needs; these are a typical set of barclamps for most SOC 6 installation:
+</para>
+      
+  <itemizedlist>
+   <listitem> 
+       <para>
+    database
+       </para>
+    </listitem>
+    
+<listitem>   
+    <para>
+rabbitmq
+</para>
+   </listitem>
+   
+<listitem>
+    <para>
+keystone
+</para>
+</listitem>
+
+<listitem>
+    <para>
+glance 
+</para>
+</listitem>
+
+<listitem>
+    <para>
+cinder
+</para>
+</listitem>
+
+<listitem>
+    <para>
+horizon
+</para>
+</listitem>
+</itemizedlist>
+  
+  <para>
+  Now you will customize the Neutron barclamp for your installation. Note that the order of <literal>ml2_type_drivers</literal> is important, and <literal>opflex</literal> always comes first. The second <literal>ml2_type_drivers</literal> option defines the encapsulation used (<literal>vxlan</literal> or <literal>vlan</literal>).  Click on the Raw button to edit the attributes in raw mode and create a proposal. Set the following values and leave everything else set to the defaults:
+  </para>
+  
+  <screen>
+{
```

Comment:
To avoid unnecessary whitespace, always put the first line of the text directly behind the &lt;screen> tag and directly follow the last line with the closing tag:

    <screen>{
    ...
    }</screen>